### PR TITLE
chore: update prettier configuration for print width

### PR DESCRIPTION
**Pull request by ajnart**
<br/>
<div align="center">
  
  <h3>Homarr</h3>
</div>

**Thank you for your contribution. Please ensure that your pull request meets the following pull request:**

- [ ] Builds without warnings or errors (``pnpm buid``, autofix with ``pnpm format:fix``)
- [ ] Pull request targets ``dev`` branch
- [ ] Commits follow the [conventional commits guideline](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] No shorthand variable names are used (eg. ``x``, ``y``, ``i`` or any abbrevation)

### CODE REVIEW

**Review by manuel-rw — DISMISSED** [review]
please format (you can do this by running ``pnpm format:fix``)

**Comment by ajnart** [comment]
Alright 😀

File: apps/websocket/wssServer.cjs
Comment (Meierschlumpf) [inline apps/websocket/wssServer.cjs:1]: This one as well should be removed

**Review by Meierschlumpf — CHANGES_REQUESTED** [review]

File: apps/tasks/tasks.cjs
[the remark this answers is not in the document]
Comment (ajnart) [reply apps/tasks/tasks.cjs:1]: Good catch

**Review by Meierschlumpf — APPROVED** [review]

**Comment by ajnart** [comment]
Thanks for the fix

**Pull request merged**